Mu2eG4/src/Mu2eG4_module.cc

```diff
@@ -322,6 +328,7 @@ namespace mu2e {
 #endif
     _runManager->SetUserInitialization(physicsList_);
 
+    _scorer->initialize();
```

Comment:
Given the potential impact on CPU and memory, could you make the use of the scorer conditional and controlled by a fhicl parameter in the prolog? Both here in the sequential mode and in the MT one?

Reply:
There is a parameter in fhicl that controls the instantiation of the scoring manager (off by default). The Mu2eG4ScoringManager is only initialized if this turned on (see Mu2eG4ScoringManager::initialize)

Reply:
ok